src/dom.js

```diff
@@ -375,6 +375,21 @@ export function childElementByTag(parent, tagName) {
 
 
 /**
+ * A wrapper around native querySelectorAll to use :scope when possible.
+ * @param {!Element} parent
+ * @param {string} selector
+ * @returns {!Array.<!Element>}
+ */
+export function querySelectorAll(parent, selector) {
```

Comment:
Definitely would call this `scopedQuerySelectorAll`

What is your concrete use case? Do you need a polyfill?